lib/archethic/metrics/poller.ex

```diff
@@ -0,0 +1,82 @@
+defmodule ArchEthic.Metrics.Poller do
+  @moduledoc """
+  Provides Telemetry of the network and Maneges
+  """
+  require Logger
+  use GenServer
+  # map because you cant pop from linked list
+  @default_state %{
+    pid_refs: %{},
+    data: ArchEthic.Metrics.Helpers.get_client_metric_default_value()
+  }
+
+  def start_link(_state) do
+    GenServer.start_link(__MODULE__, @default_state, name: __MODULE__)
+  end
+
+  def init(initial_state) do
+    periodic_metric_aggregation()
+    periodic_push_updates()
```

Comment:
-I did write docs , which would have improved understanding but nfortunately I was not able to git push with @docs @ type @specs.

-here the purpose is to provide abstraction 
     -to control polling of network though periodic _metric _ aggregation and answer How frequently you want poll the 
       network for metrics?
     -to control How frequently you want to push updates to the UI or the points on the graph.
Which shows the responsiveness  or  the graph is being actively updated.

All of these was considered under light of that

*That a normal user does not have that much patience to wait for 5 second update , As a normal user and w.r.t HCI Human computer Interaction study in my final year , it simple stats the user stays longer if its updates quickly *

Reply:
Please acknowledge with time for 
**network polling duration
Update duration**

Reply:
The thing is even if you call this `periodic_push_updates`, the client will have the same info, and the UI component may not change, and it may create more load for the node to notify and send back updates to clients.

Reply:
So I don't see a real benefit now

Reply:
But its purpose , was defined with idea that with traffic, value will change rapidly, 
With data taken  in consideration w.r.t of line chart , to create really active graph.
Yes client will do have the same info but w.r.t graphs like line chart there
will be a visible change , for the user to counter their impatience ,
 so the periodic push update was built around this concept. Its As per the HCI

Reply:
The push update is fine to notify the clients, but for me the need to re-plot the data to bring more UX should be on the UI not on the server, to avoid load. 
The node software is not just an explorer but a complete blockchain node – I think we should prioritize transaction handling than UI/UX. Having a self-hosted explorer is fine, but should not constraint the capacity for a node to behave normally.
Also, when the traffic will increase, by sticking with 5s interval the histogram will gather more aggregated metrics and will reflect more the last changes.

Reply:
Hi , 
I have removed the push updates and kept only periodic metric aggregation